xbmc/pvr/timers/PVRTimerInfoTag.cpp

```diff
@@ -40,43 +40,53 @@ using namespace EPG;
 
 CPVRTimerInfoTag::CPVRTimerInfoTag(bool bRadio /* = false */) :
   m_strTitle(g_localizeStrings.Get(19056)), // New Timer
-  m_strDirectory("/")
+  m_bFullTextEpgSearch(true)
```

Comment:
I doubt a setting is needed, people got along fine all the years that tvheadend didn't support full text matching so I believe it's safe to just default to false.